engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -1661,7 +1626,7 @@ class SemanticsObject {
     // Trivial case: previous list was empty => just populate the container.
     if (_currentChildrenInRenderOrder == null || _currentChildrenInRenderOrder!.isEmpty) {
       for (final SemanticsObject child in childrenInRenderOrder) {
-        containerElement!.append(child.element);
+        element!.append(child.element);
```

Comment:
Without the container element could we have a problem with `_scrollOverflowElement` used by scrollables? This algorithm assumes that it controls all the children of the container. However, `SemanticScrollable` appends `_scrollOverflowElement` that this algorithm is unaware of. I think we're OK, since here we only use `containerElement.append/insertBefore`. For example, we don't use `containerElement.children` or `containerElement.firstChild`. However, it might be worth paying attention to.

Reply:
So `_scrollOverflowElement` was previously a sibling of `containerElement` and is now a sibling of `children`.

I think this is OK since, as you said, we are only using `containerElement.append/insertBefore` but not `containerElement.children`.